packages/aft/lib/src/commands/test/unit_test_command.dart

```diff
@@ -0,0 +1,187 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:aft/aft.dart';
+import 'package:aft/src/test_reports/print_results.dart';
+import 'package:aft/src/test_reports/test_report_folio.dart';
+import 'package:aft/src/test_reports/test_report_scored.dart';
+import 'package:aft/src/utils/constants.dart';
+import 'package:aft/src/utils/emphasize_text.dart';
+import 'package:aft/src/utils/select_packages.dart';
+import 'package:path/path.dart' as p;
+import 'package:very_good_test_runner/very_good_test_runner.dart';
+
+/// Command to run Flutter and Dart unit tests.
+class UnitTestCommand extends AmplifyCommand {
+  @override
+  String get description => 'Runs Flutter and Dart unit tests';
+
+  @override
+  String get name => 'unit';
+
+  @override
+  Future<void> run() async {
+    final packages = await allPackages;
+
+    final testablePackages = packages
+        .where(
+          (package) => package.unitTestDirectory != null,
+        )
+        .toList();
+
+    final selectedPackages = await selectPackages(testablePackages);
+
+    if (selectedPackages.isEmpty) {
```

Comment:
Similar to the update you make in the integ test package, you can loop here and ask for input again if you don't get any the first time rather than exiting.